From a7aae58e81de44a121f1d1c4a8df43692688afbd Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Sun, 31 Jul 2011 15:03:43 +0200
Subject: [PATCH 4/4] check settings only for last selected reporter

---
 src/gui-wizard-gtk/wizard.c |   51 ++++++++++++++++++++++++++++++------------
 1 files changed, 36 insertions(+), 15 deletions(-)

diff --git a/src/gui-wizard-gtk/wizard.c b/src/gui-wizard-gtk/wizard.c
index 3dc6921..a6c48eb 100644
--- a/src/gui-wizard-gtk/wizard.c
+++ b/src/gui-wizard-gtk/wizard.c
@@ -647,28 +647,43 @@ static void analyze_rb_was_toggled(GtkButton *button, gpointer user_data)
     }
 }

-static void report_tb_was_toggled(GtkButton *button_unused, gpointer user_data_unused)
+static void report_tb_was_toggled(GtkButton *button, gpointer user_data)
 {
+    char *event_name = (char *)user_data;
     struct strbuf *reporters_string = strbuf_new();
     GList *li = g_list_reporters;
-    for (; li; li = li->next)
+
+    /* if ((button && user_data)
+     * prevents sigsegv which would happen when call from
+     * line 990: ((void (*)(GtkButton*, gpointer*))func)(NULL, NULL);
+     */
+
+    if ((button && user_data)
+        && gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(button)) == TRUE)
     {
-        event_gui_data_t *event_gui_data = li->data;
-        if (gtk_toggle_button_get_active(event_gui_data->toggle_button) == TRUE)
-        {
-            strbuf_append_strf(reporters_string,
+        if (g_list_find(li, event_name) == NULL)
+            li = g_list_prepend(li, event_name);
+
+        strbuf_append_strf(reporters_string,
                             "%s%s",
                             (reporters_string->len != 0 ? ", " : ""),
-                            event_gui_data->event_name
-            );
-            GHashTable *errors = validate_event(event_gui_data->event_name);
-            if (errors != NULL)
-            {
-                g_hash_table_unref(errors);
-                show_event_opt_error_dialog(event_gui_data->event_name);
-            }
+                            event_name
+                            );
+
+        GHashTable *errors = validate_event(event_name);
+        if (errors != NULL)
+        {
+            g_hash_table_unref(errors);
+            show_event_opt_error_dialog(event_name);
         }
+
     }
+    else
+    {
+        if (g_list_find(li, event_name) != NULL)
+            li = g_list_remove(li, event_name);
+    }
+

     gtk_assistant_set_page_complete(g_assistant,
                 pages[PAGENO_REPORTER_SELECTOR].page_widget,
@@ -796,7 +811,8 @@ static event_gui_data_t *add_event_buttons(GtkBox *box,
         }

         if (func)
-            g_signal_connect(G_OBJECT(button), "toggled", func, NULL);
+            g_signal_connect(G_OBJECT(button), "toggled", func, xstrdup(event_name));
+
         if (cfg && cfg->long_descr)
             gtk_widget_set_tooltip_text(button, cfg->long_descr);

--
1.7.6

