From 44708fe22020ca955a434f30fed55c918542c5bc Mon Sep 17 00:00:00 2001
From: Denys Vlasenko <dvlasenk@redhat.com>
Date: Wed, 10 Aug 2011 17:17:37 +0200
Subject: [PATCH 38/52] report-newt: fix help text (option -o is mandatory)

Signed-off-by: Denys Vlasenko <dvlasenk@redhat.com>
---
 src/report-newt/report-newt.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/report-newt/report-newt.c b/src/report-newt/report-newt.c
index 67ac15a..b59e31b 100644
--- a/src/report-newt/report-newt.c
+++ b/src/report-newt/report-newt.c
@@ -350,7 +350,7 @@ int main(int argc, char **argv)
 
     /* Can't keep these strings/structs static: _() doesn't support that */
     const char *program_usage_string = _(
-        "\b [-od] DIR\n"
+        "\b -o [-d] DIR\n"
         "\n"
         "newt tool to report problem saved in specified DIR"
     );
-- 
1.7.6

