From ced99d1ab56c2c5af7eaf1efbac740938126d243 Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Thu, 11 Aug 2011 18:01:04 +0200
Subject: [PATCH 46/52] uploader_event.conf: added new line to the end of file

---
 src/plugins/uploader_event.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/plugins/uploader_event.conf b/src/plugins/uploader_event.conf
index ef96c83..2406008 100644
--- a/src/plugins/uploader_event.conf
+++ b/src/plugins/uploader_event.conf
@@ -1 +1 @@
-EVENT=report_Uploader analyzer=libreport reporter-upload
\ No newline at end of file
+EVENT=report_Uploader analyzer=libreport reporter-upload
-- 
1.7.6

