From 13afd35a352d6abca5c8477c67ebd73596068c9c Mon Sep 17 00:00:00 2001
From: Nikola Pajkovsky <npajkovs@redhat.com>
Date: Fri, 12 Aug 2011 10:50:32 +0200
Subject: [PATCH 47/52] read default CONFFILE if no -c option is given

Signed-off-by: Nikola Pajkovsky <npajkovs@redhat.com>
---
 src/plugins/reporter-bugzilla.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/plugins/reporter-bugzilla.c b/src/plugins/reporter-bugzilla.c
index 43a3897..5b7d8bb 100644
--- a/src/plugins/reporter-bugzilla.c
+++ b/src/plugins/reporter-bugzilla.c
@@ -281,6 +281,8 @@ int main(int argc, char **argv)
 
     export_abrt_envvars(0);
 
+    if (!conf_file)
+        conf_file = g_list_append(conf_file, (char*) CONF_DIR"/plugins/bugzilla.conf");
     while (conf_file)
     {
         char *fn = (char *)conf_file->data;
-- 
1.7.6

