From 488f8c81ada924f8fd8d55d5d216f2c35b3f0ed5 Mon Sep 17 00:00:00 2001
From: Nikola Pajkovsky <npajkovs@redhat.com>
Date: Fri, 12 Aug 2011 14:47:12 +0200
Subject: [PATCH 51/52] fix 'make check'

Signed-off-by: Nikola Pajkovsky <npajkovs@redhat.com>
---
 po/POTFILES.in |   20 +++++++++++++++++++-
 1 files changed, 19 insertions(+), 1 deletions(-)

diff --git a/po/POTFILES.in b/po/POTFILES.in
index d0c4409..961796f 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -2,10 +2,28 @@
 # List of source files containing translatable strings.
 # Please keep this file sorted alphabetically.
 src/cli/cli.c
-src/cli/report.c
+src/cli/cli-report.c
 src/gtk-helpers/event_config_dialog.c
 src/gui-wizard-gtk/main.c
 src/gui-wizard-gtk/wizard.c
 src/gui-wizard-gtk/wizard.glade
 src/lib/event_config.c
 src/lib/parse_options.c
+src/lib/abrt_curl.c
+src/lib/client.c
+src/lib/run_event.c
+src/plugins/report.c
+src/plugins/report_Bugzilla.xml.in
+src/plugins/report_Kerneloops.xml.in
+src/plugins/report_Logger.xml.in
+src/plugins/report_Mailx.xml.in
+src/plugins/report_RHTSupport.xml.in
+src/plugins/report_Uploader.xml.in
+src/plugins/reporter-bugzilla.c
+src/plugins/reporter-kerneloops.c
+src/plugins/reporter-mailx.c
+src/plugins/reporter-print.c
+src/plugins/reporter-rhtsupport.c
+src/plugins/reporter-upload.c
+src/plugins/rhbz.c
+src/report-newt/report-newt.c
-- 
1.7.6

