--- libreport.spec.old	2012-01-21 22:32:45.222276116 +0400
+++ libreport.spec	2011-12-20 15:34:34.000000000 +0400
@@ -5,7 +5,7 @@
 Summary: Generic library for reporting various problems
 Name: libreport
 Version: 2.0.8
-Release: 4%{?dist}.R
+Release: 4%{?dist}
 License: GPLv2+
 Group: System Environment/Libraries
 URL: https://fedorahosted.org/abrt/
@@ -21,7 +21,6 @@
 Patch8: 0011-url-takes-escaped-string.patch
 Patch9: 0001-rhbz-add-the-architecture-to-the-comment-only-if-it-.patch
 Patch10: 0001-rhbz-768647-python-doen-t-have-a-backtrace_rating-fi.patch
-Patch99: %{name}-2.0.5-read-fedora-release.patch
 BuildRequires: dbus-devel
 BuildRequires: gtk2-devel
 BuildRequires: curl-devel
@@ -234,8 +233,6 @@
 %patch9 -p1
 %patch10 -p1
 
-%patch99 -p1
-
 %build
 autoconf
 %configure
@@ -397,13 +394,10 @@
 %config(noreplace) %{_sysconfdir}/libreport/events.d/uploader_event.conf
 
 %changelog
-* Tue Dec 20 2011 Nikola Pajkovsky <npajkovs@redhat.com> 2.0.8-4.R
+* Tue Dec 20 2011 Nikola Pajkovsky <npajkovs@redhat.com> 2.0.8-4
 - 768647 - [abrt] libreport-plugin-bugzilla-2.0.8-3.fc16: libreport_xatou:
            Process /usr/bin/reporter-bugzilla was killed by signal 11 (SIGSEGV)
 
-* Sat Dec 19 2011 Arkady L. Shane <ashejn@russianfedora.ru> 2.0.8-3.R
-- push Fedora instead of RFRemix to bugzilla.
-
 * Fri Dec 09 2011 Jiri Moskovcak <jmoskovc@redhat.com> 2.0.8-3
 - fixed few crashes in bodhi plugin
 
