diff -urN libreport-2.0.13.orig/src/lib/dump_dir.c libreport-2.0.13/src/lib/dump_dir.c
--- libreport-2.0.13.orig/src/lib/dump_dir.c	2012-09-13 12:18:32.774086071 +0400
+++ libreport-2.0.13/src/lib/dump_dir.c	2012-09-13 12:20:27.557294225 +0400
@@ -553,7 +553,7 @@
 
     if (!release)
     {
-        release = load_text_file("/etc/system-release",
+        release = load_text_file("/etc/fedora-release",
                 DD_LOAD_TEXT_RETURN_NULL_ON_FAILURE);
         if (!release)
             release = load_text_file("/etc/redhat-release", /*flags:*/ 0);
@@ -561,7 +561,7 @@
         if (chroot_dir)
         {
             free(release);
-            char *chrooted_name = concat_path_file(chroot_dir, "/etc/system-release");
+            char *chrooted_name = concat_path_file(chroot_dir, "/etc/fedora-release");
             release = load_text_file(chrooted_name, /*flags:*/ 0);
             free(chrooted_name);
             if (release[0])
