diff -urN libreport-2.0.3.orig/src/lib/dump_dir.c libreport-2.0.3/src/lib/dump_dir.c
--- libreport-2.0.3.orig/src/lib/dump_dir.c	2011-06-24 17:19:45.208289959 +0400
+++ libreport-2.0.3/src/lib/dump_dir.c	2011-06-24 17:20:37.903246846 +0400
@@ -493,10 +493,10 @@
     dd_save_text(dd, FILENAME_ARCHITECTURE, buf.machine);
     dd_save_text(dd, FILENAME_HOSTNAME, buf.nodename);
 
-    char *release = load_text_file("/etc/system-release",
+    char *release = load_text_file("/etc/fedora-release",
                 DD_LOAD_TEXT_RETURN_NULL_ON_FAILURE);
     if (!release)
-        release = load_text_file("/etc/redhat-release", /*flags:*/ 0);
+        release = load_text_file("/etc/fedora-release", /*flags:*/ 0);
     dd_save_text(dd, FILENAME_OS_RELEASE, release);
     free(release);
 }
diff -urN libreport-2.0.3.orig/src/report-python/__init__.py libreport-2.0.3/src/report-python/__init__.py
--- libreport-2.0.3.orig/src/report-python/__init__.py	2011-06-24 17:19:45.187289579 +0400
+++ libreport-2.0.3/src/report-python/__init__.py	2011-06-24 17:21:02.276689438 +0400
@@ -19,7 +19,7 @@
 
 import os
 
-SYSTEM_RELEASE_PATHS = ["/etc/system-release","/etc/redhat-release"]
+SYSTEM_RELEASE_PATHS = ["/etc/fedora-release","/etc/system-release","/etc/redhat-release"]
 ####SYSTEM_RELEASE_DEPS = ["system-release", "redhat-release"]
 
 _hardcoded_default_product = ""
