diff -urN libreport-2.0.5.orig/src/lib/dump_dir.c libreport-2.0.5/src/lib/dump_dir.c
--- libreport-2.0.5.orig/src/lib/dump_dir.c	2011-09-17 20:34:03.503691016 +0400
+++ libreport-2.0.5/src/lib/dump_dir.c	2011-09-17 20:37:35.645374073 +0400
@@ -510,10 +510,10 @@
     if (release)
         return;
 
-    release = load_text_file("/etc/system-release",
+    release = load_text_file("/etc/fedora-release",
             DD_LOAD_TEXT_RETURN_NULL_ON_FAILURE);
     if (!release)
-        release = load_text_file("/etc/redhat-release", /*flags:*/ 0);
+        release = load_text_file("/etc/fedora-release", /*flags:*/ 0);
     dd_save_text(dd, FILENAME_OS_RELEASE, release);
     free(release);
 }
diff -urN libreport-2.0.5.orig/src/report-python/__init__.py libreport-2.0.5/src/report-python/__init__.py
--- libreport-2.0.5.orig/src/report-python/__init__.py	2011-09-17 20:34:03.492691036 +0400
+++ libreport-2.0.5/src/report-python/__init__.py	2011-09-17 20:38:31.062066091 +0400
@@ -21,8 +21,8 @@
 
 import os
 
-SYSTEM_RELEASE_PATHS = ["/etc/system-release","/etc/redhat-release"]
-SYSTEM_RELEASE_DEPS = ["system-release", "redhat-release"]
+SYSTEM_RELEASE_PATHS = ["/etc/fedora-release", "/etc/system-release","/etc/redhat-release"]
+SYSTEM_RELEASE_DEPS = ["fedora-release", "system-release", "redhat-release"]
 
 _hardcoded_default_product = ""
 _hardcoded_default_version = ""
